lib/commands.py

```diff
@@ -546,6 +547,20 @@ def _mktx(self, outputs, fee=None, change_addr=None, domain=None, nocheck=False,
                 self.wallet.save_transactions()
         return tx
 
+    @command('')
+    def rpa_generate_paycode(self):
+        return rpa_paycode.rpa_generate_paycode(self.wallet)
+
+    @command('wp')
+    def rpa_generate_transaction_from_paycode(self, amount,  paycode=None, fee= None, from_addr=None, change_addr=None, nocheck=False, unsigned=False, password=None, locktime=None,
```

Comment:
Weird formatting on args.  Some have double-spaces after comma, some have space after = and some don't.  Please fix...

Reply:
i fixed but wasn't sure the proper way to do spacing when the args wrap onto a new line.  Hope this is ok.

Reply:
Well.. this is why I recommend `PyCharm`: https://www.jetbrains.com/pycharm/

It's the bee's knees.  I really teaches you all the dumb stuff like the Python formatting nits that are popular and it catches errors. You become enlightened Python being by using it that emanates and radiates Python energy into the universe.  It's good stuff.  Please install it!!

Reply:
And yes the new commit is correct FYI.